lnclient/ldk/ldk.go

```diff
@@ -1071,15 +1071,27 @@ func (ls *LDKService) GetOnchainBalance(ctx context.Context) (*lnclient.OnchainB
 		}
 	}
 
+	pendingSweepBalanceDetails := make([]lnclient.PendingBalanceDetails, 0)
+	increasePendingBalanceFromClosure := func(nodeId, channelId *string, amount uint64, fundingTxId *ldk_node.Txid, fundingTxIndex *uint16) {
+		pendingBalancesFromChannelClosures += amount
+		pendingSweepBalanceDetails = append(pendingSweepBalanceDetails, lnclient.PendingBalanceDetails{
+			NodeId:        utils.UnwrapOrZero(nodeId),
```

Comment:
If we do it this way we can add an invalid `lnclient.PendingBalanceDetails`. Can we instead check all the properties are not nil, and only if so add the element to the array?